src/main.c

```diff
@@ -29,6 +29,7 @@ int	main(int argc, char **argv)
 		return (1);
 	}
 	ft_cam_init(&render.cam, render.mlx_ptrs.img.size);
+	ft_init_menu(&render.menu, MENU_OPACITY, MENU_BACK, MENU_FONT);
```

Comment:
Would call it ft_menu_init() or rename ft_init_cam()

Reply:
Renamed to ft_menu_init()